src/ZipkinOpenTracing/Tracer.php

```diff
@@ -38,6 +37,59 @@ public function __construct(ZipkinTracing $tracing)
     {
         $this->tracer = $tracing->getTracer();
         $this->propagation = $tracing->getPropagation();
+        $this->scopeManager = new ScopeManager();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getScopeManager()
+    {
+        return $this->scopeManager;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getActiveSpan()
+    {
+        $activeScope = $this->scopeManager->getActiveScope();
+        if ($activeScope === null) {
+            return null;
+        }
+
+        return $activeScope->getSpan();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function startActiveSpan($operationName, $options = [])
+    {
+        if (!$options instanceof SpanOptions) {
+            $options = SpanOptions::create($options);
+        }
+
+        $child = $this->findChildInReferences($options->getReferences());
+        if ($child === null && $this->getActiveSpan() !== null) {
+            $child = $this->getActiveSpan()->getContext();
+        }
+
+        $options = SpanOptions::create(array_merge([
```

Comment:
This is something I want to avoid and it is something we should foresee. This is one of the main reasons for me to finish this: https://github.com/opentracing/opentracing-php/pull/46

Reply:
@eligijusvitkauskas-home24 now you can use https://github.com/opentracing/opentracing-php/blob/master/src/OpenTracing/SpanOptions.php#L110